README.md

```diff
@@ -9,23 +9,69 @@
 
 ---
 
-Crane (FinOps Crane) is a cloud native open source project which manages cloud resources on Kubernetes stack, it is inspired by FinOps concepts.
+## What is Crane
 
-## Introduction
+> English | [中文](README_zh.md)
 
-The goal of Crane is to provide a one-stop-shop project to help Kubernetes users to save cloud resource usage with a rich set of functionalities:
+Crane is a full lifecycle FinOps Platform for Cloud Resource Analytics and Economics in Kubernetes clusters.The goal is not only help user to manage cloud cost easier but also ensure the quality of applications.
 
-- **Time Series Prediction** based on monitoring data
-- **Usage and Cost visibility**
-- **Usage & Cost Optimization** including:
-  - R2 (Resource Re-allocation)
-  - R3 (Request & Replicas Recommendation)
-  - Effective Pod Autoscaling (Effective Horizontal & Vertical Pod Autoscaling)
-  - Cost Optimization
-- **Enhanced QoS** based on Pod PriorityClass
-- **Load-aware Scheduling** 
+**How to start a Cost-Saving journey on Crane?**
 
-<img alt="Crane Overview" height="550" src="docs/images/crane-overview.png" width="800"/>
+1. **Visualization**: Cost insight for cloud assets and kubernetes resources(Deployments, StatefulSets).
```

Comment:
Visualization is one way of `Understanding` cost, so is `Understanding` better fit here?